pyproject.toml

```diff
@@ -1,7 +1,80 @@
 [build-system]
-requires = [ "setuptools", "wheel" ]
+requires = ["setuptools >= 63.0.0"]
```

Comment:
At some point, maybe not now, we might want [`hatchling`](https://hatch.pypa.io/1.9/) as a build backend instead, because it's easier to hook into, and we could easily ensure the resulting wheel has the binary component included.

Reply:
Agreed. I've only worked with Python for a few years now, but my general impression is that the packaging ecosystem leaves much to be desired. No idea what the best tool is, but for right now personally I favor smaller incremental changes over bigger ones.